cmd/cli/uninstall_mesh.go

```diff
@@ -141,6 +141,24 @@ func (d *uninstallMeshCmd) run() error {
 		}
 
 		if err == nil {
+			namespaces, _ := selectNamespacesMonitoredByMesh(d.meshName, d.clientSet)
+			if len(namespaces.Items) != 0 {
+				for _, ns := range namespaces.Items {
+					removeCmd := &namespaceRemoveCmd{
+						out:       d.out,
+						namespace: ns.Name,
+						meshName:  d.meshName,
+						clientSet: d.clientSet,
+					}
+
+					err := removeCmd.run()
+
+					// Unable to remove namespace in mesh
+					if err != nil {
+						return err
```

Comment:
I wonder if it would be helpful to log the reason for the inability to remove namespaces with `log.Error().Err(err).Msgf("Error removing namespaces: %+v", namespaces.Items)`
OR we could assume that the caller would do that for us based on the `err` they get.

Reply:
I believe caller one is good.
what do other people suggest?

Reply:
Can we mark this as resolved @draychev?